app/vmauth/main.go

```diff
@@ -41,6 +41,7 @@ var (
 	reloadAuthKey        = flag.String("reloadAuthKey", "", "Auth key for /-/reload http endpoint. It must be passed as authKey=...")
 	logInvalidAuthTokens = flag.Bool("logInvalidAuthTokens", false, "Whether to log requests with invalid auth tokens. "+
 		`Such requests are always counted at vmauth_http_request_errors_total{reason="invalid_auth_token"} metric, which is exposed at /metrics page`)
+	brokenBackendBackoff = flag.Duration("brokenBackendBackoff", 3*time.Second, "Sets a delay period for load balancing to skip a malfunctioning backend. (defaults 3s)")
```

Comment:
```suggestion
	brokenBackendBackoff = flag.Duration("brokenBackendBackoff", 3*time.Second, "Sets a delay period for load balancing to skip a malfunctioning backend.
```
There is no need to add default values here, go flags package automatically appends default values to the flag description.

Reply:
Thanks!